src/test/java/build/buildfarm/common/redis/BalancedRedisQueueMockTest.java

```diff
@@ -359,7 +362,7 @@ public void canQueueIfiniteQueueAllowsQueuing() throws Exception {
   @Test
   public void canQueueFullQueueNotAllowsQueuing() throws Exception {
     // MOCK
-    when(redis.llen(any(String.class))).thenReturn(123L);
```

Comment:
I think these changes are only valid when the queue is configured as ""priority",  
The default is "regular", and it can only be changed by changing the code.  

We'll run the redis tests in the CI next and that will clear up any differences.

Reply:
Yeah, I fixed that. We now can speciffy through BalancedRedisQueue constructor witch queue to use